web/script.js

```diff
@@ -115,7 +121,7 @@
                 </Element>\
               </StyledText>\
             </TextObject>\
-            <Bounds X="270" Y="2496" Width="2790" Height="384" />\
+            <Bounds X="270" Y="1920" Width="2790" Height="672" />\
```

Comment:
Was just wondering since you changed the Y and Height dimensions. There are label printers in the TechX office. You need to install the driver on your computer as well. @anishathalye probably knows more about them